docs/presentations/articles/OS_LLMs_Robert.md

```diff
@@ -0,0 +1,278 @@
+<span style="color:red; font-size: 40px; font-weight: bold;">Open-Source LLMs</span>
+
+#### An Overview of Open Source LLMs: What Open Source Alternatives Exist to OpenAI's GPT-3.5 and How Can I Utilize Them?
```

Comment:
no need for a subtitle. This will look strange in the outline:

![image](https://github.com/pkeilbach/htwg-practical-nlp/assets/28002199/9fea4d0c-6cf8-4809-aef8-82e4975d14ef)

```suggestion
This article gives an overview of Open Source LLMs: what Open Source alternatives exist to OpenAI's GPT-3.5 and how can we utilize them?
```

Reply:
thanks